django_ip_geolocation/backends/base.py

```diff
@@ -16,6 +16,8 @@ def _parse(self):
         raise NotImplementedException()
 
     def data(self):
```

Comment:
What is this `data` method doing? why can't you replace it with parse directly ? (this way the user returns what he wants and you don't have the problem with 'internal' methods anymore...)

Reply:
The data method is here to return the "standard" format that I specified. 

The user shouldn't return what he wants. If a user decides to change the backend, it should work without changing anything in his app.